tests/Doctrine/Tests/Models/ReadonlyProperties/Book.php

```diff
@@ -5,7 +5,6 @@
 namespace Doctrine\Tests\Models\ReadonlyProperties;
 
 use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\Common\Collections\Collection;
```

Comment:
This as been automatically fixed by `vendor/bin/phpcbf` (and flagged by `vendor/bon/phpcs`). The original error was not related to this PR and was present before any changes were made.

```
# vendor/bin/phpcs 
.....................................E...................... 60 / 76 (79%)
................                                             76 / 76 (100%)



FILE: tests/Doctrine/Tests/Models/ReadonlyProperties/Book.php
--------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------
 8 | ERROR | [x] Type Doctrine\Common\Collections\Collection is not used in this file.
   |       |     (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
--------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------------

Time: 998ms; Memory: 174MB
```

```
# vendor/bin/phpcbf 
.................F.......................................... 60 / 76 (79%)
................                                             76 / 76 (100%)



PHPCBF RESULT SUMMARY
----------------------------------------------------------------------------
FILE                                                        FIXED  REMAINING
----------------------------------------------------------------------------
tests/Doctrine/Tests/Models/ReadonlyProperties/Book.php     1      0
----------------------------------------------------------------------------
A TOTAL OF 1 ERROR WERE FIXED IN 1 FILE
----------------------------------------------------------------------------

Time: 12,03 secs; Memory: 10MB
```



I'm happy to reverse if this is outside of the scope of this PR.

Reply:
Yes, please revert. And please use PHP 8.1 to execute PHPCS.

Reply:
@derrabus reverted, thanks !